workspaces/servicenow/plugins/servicenow-backend/config.d.ts

```diff
@@ -52,5 +31,13 @@ export interface ServiceNowConfig {
      * @visibility secret
      */
     oauth?: OAuthConfig;
+    /** The base url of the ServiceNow instance.
+     * @visibility backend
+     */
+    cmdbBaseUrl: string;
```

Comment:
Can you explain why another baseurl is needed? Is this something different then the `instanceUrl`?